static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -148,13 +219,83 @@ class Issues extends Component<Props, State> {
           path: `/organizations/${organization.slug}/issues/`,
           queryParams: {
             ...queryParams,
-            query: new QueryResults([`first-release:${version}`]).formatString(),
+            query: new QueryResults([`${IssuesQuery.NEW}:${version}`]).formatString(),
           },
         };
     }
   }
 
+  async fetchIssuesCount() {
+    const {api, organization, version} = this.props;
+    const issueCountEndpoint = this.getIssueCountEndpoint();
+    const resolvedEndpoint = `/organizations/${organization.slug}/releases/${version}/resolved/`;
+
+    try {
+      await Promise.all([
+        api.requestPromise(issueCountEndpoint),
+        api.requestPromise(resolvedEndpoint),
+      ]).then(([issueResponse, resolvedResponse]) => {
+        this.setState({
+          count: {
+            all: issueResponse[`${IssuesQuery.ALL}:${version}`],
+            new: issueResponse[`${IssuesQuery.NEW}:${version}`],
+            resolved: resolvedResponse.length,
+            unhandled:
+              issueResponse[`${IssuesQuery.UNHANDLED} ${IssuesQuery.ALL}:${version}`],
```

Comment:
Let's fallback to `0` because this endpoint returns `null` in case of no issues and we want to see `0` in the buttonbar.

```suggestion
            all: issueResponse[`${IssuesQuery.ALL}:${version}`] || 0,
            new: issueResponse[`${IssuesQuery.NEW}:${version}`] || 0,
            resolved: resolvedResponse.length,
            unhandled:
              issueResponse[`${IssuesQuery.UNHANDLED} ${IssuesQuery.ALL}:${version}`] || 0,
```